investor_wallet_platform_base/models/res_partner.py

```diff
@@ -285,16 +287,26 @@ def _compute_industry_char_list(self):
             partner.industry_char_list = partner.industry_id.mapped("full_name")
 
     @api.multi
+    @api.depends(
+        "share_type_ids",
+        "share_type_ids.display_on_website",
+        "share_type_ids.state",
+        "loan_issue_ids",
+        "loan_issue_ids.display_on_website",
+        "loan_issue_ids.state",
```

Comment:
`"share_type_ids"` and `"loan_issue_ids"` are not needed: only the deepest fields are needed.